osgp/platform/osgp-secret-management/src/integration-test/java/org/opensmartgridplatform/secretmanagement/application/SoapServiceSecretManagementIT.java

```diff
@@ -325,9 +326,12 @@ public void generateAndStoreSecretsAlreadyNewSecretPresent() throws IOException
     // Generate and store secret: this should NOT result in a fault message
     final Resource generateAndStoreRequest =
         new ClassPathResource("test-requests/generateAndStoreSecrets.xml");
-    this.mockWebServiceClient
-        .sendRequest(withSoapEnvelope(generateAndStoreRequest))
-        .andExpect(ResponseMatchers.noFault());
+    final ResponseActions response =
+        this.mockWebServiceClient
+            .sendRequest(withSoapEnvelope(generateAndStoreRequest))
+            .andExpect(ResponseMatchers.noFault());
+
+    assertThat(response);
```

Comment:
Revert to original code:
```suggestion
    this.mockWebServiceClient
        .sendRequest(withSoapEnvelope(generateAndStoreRequest))
        .andExpect(ResponseMatchers.noFault());
```
and mark the issue in sonarqube as either `False positive` or `Won't fix`.